src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  fahrenheit: 60,
+  city: 'Edmonds',
+  sky: 'Sunny',
+};
+let num = state.fahrenheit;
+const tempUp = (event) => {
+  state.fahrenheit += 1;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const increase = document.getElementById('f');
+  increase.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return increase.textContent;
+};
+
+const tempDown = (event) => {
+  state.fahrenheit--;
+  tempChangeColor(state.fahrenheit);
+  weatherGarden();
+  const decrease = document.getElementById('f');
+  decrease.textContent = `Degrees Fahrenheit:${state.fahrenheit}`;
+  return decrease.textContent;
+};
+
+const cityInput = (event) => {
```

Comment:
The method name `getCityWeather` on line 30 is very descriptive because it tells me exactly what the method will do. The name `cityInput` is vague here. If you refactor this method from `cityInput` to `updateCityInputAndGetRealWeather` then you increase readability overall because someone could read the method name and know what the method will do.